benchexec/runexecutor.py

```diff
@@ -656,7 +663,7 @@ def _setup_ulimit_time_limit(self, hardtimelimit, cgroups):
     def execute_run(self, args, output_filename, stdin=None,
                    hardtimelimit=None, softtimelimit=None, walltimelimit=None,
                    cores=None, memlimit=None, memory_nodes=None,
-                   environments={}, workingDir=None, maxLogfileSize=None,
+                   environments={}, energy_measurement=None, workingDir=None, maxLogfileSize=None,
```

Comment:
I understand.

The old code is not relevant anymore. In the future, we will probably want more flexibility in order to allow more measurements, but we don't know yet how this should look, so we can just do this later.

In any case, this should probably be a parameter to the constructor, and not to `execute_run`, and appended to the parameter list for backwards compatibility (`RunExecutor` is public API).

So, please, for now just hard-code this.